src/stateMachines/wifiNetworkConfiguration.ts

```diff
@@ -29,6 +29,8 @@ export interface WiFiConfigContext {
   generalSettings: AMT.Models.GeneralSettings
   wifiSettings: any
   wifiEndPointSettings?: any
+  wifiProfileName: string
+  targetAfterError: string
```

Comment:
targetAfterError is not used? not sure why it was added? (I know it is present in other machines and probably also not used, I think it is currently only used in activation to 'go back' to another state on and UnexpectedParseError)